src/main/reactjs/src/components/contactRequest/ContactRequestFormSteps.tsx

```diff
@@ -230,6 +249,7 @@ export const WriteMessageStep = ({
         <div className="rows gapped">
           <H3>{t('steps.' + stepsByIndex[2])}</H3>
           <TextField
+            id="contact-request-form-message-field"
```

Comment:
Käytetäänkö tätä jossain? Jos se on testausta varten voisi käyttää data-testid attribuuttia.

Reply:
TextFieldeille löytyy id:t, jotta DOMissa niiden labelit osataan kohdistaa oikein (`<label for='textfieldin-id-attribuutti' ../>`). https://mui.com/components/text-fields/#accessibility

Sitä labelia käytetään toki tämän PR:n yhteydessä vain testien selektorina, mutta muutenkin saavutettavuuden kannalta pitäisi olla hyvä.